### PR TITLE
Resize WebView widget once the loginpage rendered

### LINKED ISSUE #5160: [Bug]: Login dialogue too small, not correctly displaying login page with WebView
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

The login dialogue has not the sufficcent size to correctly display the login page in itself using WebView.

### Steps to reproduce

1. Start the login process
2. When not using Flow2 for the login the dialogue is not resized according to somewhat the size of the login page
login_integrated_webengineview

### Expected behavior

The dialogue should show at least a larger amount of the login page when displaying it itself using WebView.
login_integrated_webengineview_resized

### Which files are affected by this bug

src/gui/wizard/webview.ui, src/gui/wizard/webview.cpp, src/gui/creds/webflowcredentialsdialog.cpp, src/gui/creds/webflowcredentials.cpp

### Operating system

Linux

### Which version of the operating system you are running.

openSUSE Tumbleweed

### Package

Compiled it myself

### Nextcloud Server version

23.0.11

### Nextcloud Desktop Client version

master

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by xllndr**
When the login dialogue displays the login page itself the WebView widget did not resize according to
the dimensions of the login page.

Fixes #5160

### CODE REVIEW

**Review by claucambra — APPROVED** [review]
Nice, thanks for the contribution!

**Comment by claucambra** [comment]
@xlindr could you remove the merge commits? once these are removed we will be able to merge :)

**Comment by xllndr** [comment]
Hello @claucambra, I just pushed again without any merge commits, now you should be able to merge into master, thank you!

**Comment by xllndr** [comment]
Hello @claucambra is there anyway I can help to speed up the merge into upstream so the changes get into the next version? This fix is highly appreciated by some people.

**Review by claucambra — APPROVED** [review]

**Comment by xllndr** [comment]
Hello @claucambra thank you for acting quickly last time I commented. What can I do to help this PR to be merged for the next release? Or can I reach out to someone else?

**Comment by claucambra** [comment]
> Hello @claucambra thank you for acting quickly last time I commented. What can I do to help this PR to be merged for the next release? Or can I reach out to someone else?

Yeah, sorry for losing track of this PR

Will merge once pipeline is complete :)

**Comment by xllndr** [comment]
Thank you very much @claucambra

**Comment by claucambra** [comment]
> Thank you very much @claucambra 

No worries, thanks for your contribution!

**Pull request merged**
